packages/block-editor/src/components/iframe/index.js

```diff
@@ -45,6 +45,14 @@ function bubbleEvent( event, Constructor, frame ) {
 	}
 
 	const newEvent = new Constructor( event.type, init );
+	// This is a hacky way of keeping the original "target" of the event
+	// when calling dispatchEvent on the frame. This is needed because
+	// if the target is set to the iframe, some keyboard shortcuts may trigger
+	// inadvertantly (like global shortcuts when typing within inputs...)
+	Object.defineProperty( newEvent, 'target', {
+		writable: false,
+		value: event.target,
```

Comment:
I guess that makes sense, the React events also have the original target

Reply:
But it doesn't work unfortunately. It breaks shortcuts like the "duplicate block" one and I'm not sure how to fix.

Reply:
I think we should re-explore the provider tbh, and avoid this bubbling entirely.

Reply:
I agree that there's something to be reconsidered here but I don't know yet what the solution is. I think a provider like we used to have is a hack. It relies on React event bubbling working but not DOM events) which means we won't be able to have global shortcuts). 

I'm thinking maybe we could have a special provider wrapping the iframe body content, but I'm not entirely sure how that would work.